packages/react-vapor/src/components/checkbox/examples/CheckboxExamples.tsx

```diff
@@ -24,15 +26,16 @@ export class CheckboxExamples extends React.Component<any, ICheckboxWithStateSta
 
     render() {
         return (
-            <div className="mt2">
-                <h1 className="text-medium-blue mb1 bold">Checkboxes</h1>
-                <p className="m2">
+            <div className="">
+                <p className="pb2">
                     See{' '}
                     <a href="http://coveo.github.io/vapor/controls/#checkboxes-infos" target="_blank">
                         Usage and guidelines
                     </a>
                 </p>
                 <div className="form-group">
+                    <label className="form-control-label">Checkbox set</label>

```

Comment:
Wish you could use the new `FormGroup` and `LabeledValue` here 😄

Reply:
I know, right!?
I almost started to integrate it and remembered that it was not quite ready yes xD .

We will include it everywhere as soon as we can!